src/documentation/system_administrators/advanced/quantum_safe_storage/quantum_safe_storage_deploy.md

```diff
@@ -0,0 +1,594 @@
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+ProtectHome=true
+ProtectSystem=true
+ReadWritePaths=/data /var/log
+ExecStart=/bin/zstor \
+  --log_file /var/log/zstor.log \
+  -c /etc/zstor-default.toml \
+  monitor
+Restart=always
+RestartSec=100ms
+TimeoutStopSec=5m
+
+[Install]
+WantedBy=multi-user.target
+```
+
+#### Zstor Zinit
+
+```sh
+$EDITOR /etc/zinit/zstor.yaml
+```
+
+```yaml
+exec: /bin/zstor \
+  --log_file /var/log/zstor.log \
+  -c /etc/zstor-default.toml \
+  monitor
+shutdown_timeout: 300
+```
+
+### Zdb
+
+Next is zdb. There are two arguments here that might be of interest for tuning. The first is `--datasize`, which is the maximum size of data blocks, in bytes. Here use 64MiB.
+
+The other argument to consider is `--rotate`, which is the time at which incomplete data blocks are closed and backed up. This value is in seconds, so the example is 15 minutes. Reducing this time can help reduce the chance of data loss if the front end is lost, but it will also result in more data fragmentation which can impact performance
+
+In this case, we use a shorter `TimeoutStopSec`, to give some time for zdb to flush remaining data to disk, but with the assumption that this happens much more quickly than zstor's network based operations.
+
+#### Zdb Systemd
+
+```sh
+$EDITOR /etc/systemd/system/zdb.service
+```
+
+```
+[Unit]
+Wants=network.target zstor.service
+After=network.target zstor.service
+
+[Service]
+ProtectHome=true
+ProtectSystem=true
+ReadWritePaths=/data /var/log
+ExecStart=/usr/local/bin/zdb \
+    --index /data/index \
+    --data /data/data \
+    --logfile /var/log/zdb.log \
+    --datasize 67108864 \
+    --hook /usr/local/bin/zdb-hook.sh \
+    --rotate 900
+Restart=always
+RestartSec=5
+TimeoutStopSec=60
+
+[Install]
+WantedBy=multi-user.target
+```
+
+#### Zdb Zinit
+
+```sh
+$EDITOR /etc/zinit/zdb.yaml
+```
+
+```yaml
+exec: /usr/local/bin/zdb \
+    --index /data/index \
+    --data /data/data \
+    --logfile /var/log/zdb.log \
+    --datasize 67108864 \
+    --hook /usr/local/bin/zdb-hook.sh \
+    --rotate 900
+shutdown_timeout: 60
+after: zstor
+```
+
+### Zdbfs
+
+And finally, zdbfs. The only option to configure here is the mount point. For this example we use `/mnt/qsfs`. Remember that if you are using a different mount point, it's best to also update it in the zstor config file.
+
+In the case of zdbfs, the stop timeouts are less relevant. When zdbfs receives `TERM` it will exit regardless of any ongoing writes, and those write operations will encounter an error. There is a final flush to the zdb, but this happens very quickly when the zdb is running on the same machine so five seconds should be more than enough.
+
+Note that upon subsequent starts when using the `autons` option, zdbfs will give some non fatal errors about not being able to create zdb namespaces. These are really just informational messages that can be ignored.
+
+#### Zdbfs Systemd
+
+```
+$EDITOR /etc/systemd/system/zdb.service
```

Comment:
Because it's repeated here again

Reply:
But this one should indeeds be `zdbfs.service`. Will update, thanks.